vscode/src/custom-prompts/utils/menu.ts

```diff
@@ -10,8 +10,11 @@ export const NewCustomCommandConfigMenuOptions = {
 }
 
 const inlineSeparator: QuickPickItem = { kind: -1, label: 'inline' }
-const chatOption: QuickPickItem = { label: 'Ask a Question' }
+const chatOption: QuickPickItem = { label: 'Explain code' }
```

Comment:
```suggestion
const chatOption: QuickPickItem = { label: 'Ask a Question' }
```
the ''Ask a Question'' option is for creating a new inline chat from the menu